src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Program.Main.cs

```diff
@@ -140,11 +140,6 @@ public static void Main(string[] args)
 
         app.UseRouting();
 
```

Comment:
There seem to be missing top-level statement related changes for RazorPages/StarterWeb and WebAPI